javascriptsource/nanoflowcommons/actions/dependencies.txt

```diff
@@ -75,6 +75,15 @@ ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE
 OR OTHER DEALINGS IN THE SOFTWARE.
 
 
+---
+
+Name: js-base64
+Version: 3.7.2
+License: BSD-3-Clause
+Private: false
+Description: Yet another Base64 transcoder in pure-JS
+Repository: undefined
+
```

Comment:
I think we should include original copyright from people who created this lib and also the license text itself.